x-pack/filebeat/module/kubernetes/_meta/fields.yml

```diff
@@ -0,0 +1,134 @@
+- key: kubernetes
+  title: Kubernetes
+  description: >
+    Module for handling logs produced by Kubenetes pods
+  fields:
+  - name: coredns
+    type: group
+    description: >
+      Fields from coredns logs after normalization
+    fields:
+    - name: duration
+      type: double
+
+    - name: client
```

Comment:
This field should probably be placed in `source.ip`. Please take a look to ECS recommendations https://github.com/elastic/ecs

Reply:
The field is indeed normalized to be copied to source.address and source.ip in the ingest pipeline.